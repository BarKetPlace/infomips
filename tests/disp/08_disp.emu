#-------
#TEST_RETURN_CODE=FAIL           
#TEST_COMMENT="Highest adress"
#-------


load ./elf/boucle.o
disp mem 0xffffff00:0xffffff0
