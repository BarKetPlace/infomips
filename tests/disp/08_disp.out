0x14000000	0xfc000000	BNE	I	3	rs rt imm 
0x00000000	0xffffffff	NOP	J	0	
0x08000000	0xfc000000	J	J	1	target 
0x20000000	0xfc000000	ADDI	I	3	rt rs imm 
0x00000020	0xfc0007ff	ADD	R	3	rd rs rt 
